plugins/circleci/package.json

```diff
@@ -46,7 +46,9 @@
     "@types/jest": "^26.0.7",
     "@types/node": "^12.0.0",
     "@types/react-lazylog": "^4.5.0",
-    "jest-fetch-mock": "^3.0.3"
+    "jest-fetch-mock": "^3.0.3",
+    "msw": "^0.20.5",
```

Comment:
these deps don't make sense for frontend plugins right?

Reply:
@benjdlambert I think this fix is to stop the plugin: diff command to stop complaining but I think it needs to be changed to compare frontend and backend plugin separately

Reply:
I want to standardise that anyway. `jest-fetch-mock` will disappear in favour of `node-fetch` and `msw` for both backend and frontend plugins. All the tests run in `node` so it doesn't matter either way.